core/operation/src/main/java/uk/gov/gchq/gaffer/jobtracker/JobDetail.java

```diff
@@ -49,8 +49,8 @@ public JobDetail(final JobDetail oldJobDetail, final JobDetail newJobDetail) {
         }
     }
 
-    public JobDetail(final String jobId, final String userId, final OperationChain<?> opChain, final JobStatus jobStatus, final String description) {
-        final String opChainString = null != opChain ? opChain.toString() : "";
+    public JobDetail(final String jobId, final String userId, final Operation operation, final JobStatus jobStatus, final String description) {
```

Comment:
Do we need to change the OperationChain parameter to Operation? If possible we should leave it as OperationChain. Otherwise we would need to change the opChain field to operation. Then any previous job details stored in the cache would probably become invalid as they won't be able to be deserialised.